### PR TITLE
move all configuration items to new config class

### PR DESCRIPTION
**Pull request by NotAdam**

### CODE REVIEW

File: config/global.ini.default [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+Database = sapphire
+Username = sapphire
+Password =
+SyncThreads = 2
+AsyncThreads = 2
+
+[Parameters]
+ServerSecret = default
+DataPath = C:\\SquareEnix\\FINAL FANTASY XIV - A Realm Reborn\\game\\sqpack
+WorldID = 67
```
Comment (SapphireMordred) [inline config/global.ini.default]: this should be in world

Comment (NotAdam) [reply config/global.ini.default]: it's needed for lobby/world

Comment (SapphireMordred) [reply config/global.ini.default]: Ah, fair enough. We shall revisit it if we decide to allow multiple worlds.

Comment (NotAdam) [reply config/global.ini.default]: yeah, I only moved it into global because we using it heaps in housing and we've hardcoded it everywhere

Comment (NotAdam) [reply config/global.ini.default]: speaking of which, going to rename this to general

**Pull request merged**